api.js

```diff
@@ -101,16 +101,24 @@ const serverOptions = {
             };
 
             Object.keys(req.params || {}).forEach(key => {
-                let value = typeof req.params[key] === 'string' ? req.params[key] : util.inspect(req.params[key], false, 3).toString().trim();
+                // cast value to string, as we want to have value present whenever there is some actual value (whatever the data type)
```

Comment:
I think it should be the reverse. If the value is `false`, `null` or empty string, then this key should be ignored and there is no need to log it. This way we can avoid key specific exceptions.

Reply:
Ignore this comment, did not realize this PR was already closed. See the same comment in active PR